### DISCUSSION TITLE
Missing linux/amd64 for 3.12.8-management-alpine

### DISCUSSION THREAD

**Discussion by torreytsui** [docker-library/rabbitmq#674] [Q&A] [answered]
```
docker run rabbitmq:3.12.8-management-alpine       

3.12.8-management-alpine: Pulling from library/rabbitmq
docker: no matching manifest for linux/amd64 in the manifest list entries.
```

While `3.12-management-alpine` for amd64 is pushed (available)

![Screenshot 2023-11-01 at 11 31 24](https://github.com/docker-library/rabbitmq/assets/945819/b45bdd8a-eb01-4d0b-85c4-e44806d54699)
![Screenshot 2023-11-01 at 11 30 09](https://github.com/docker-library/rabbitmq/assets/945819/b0a6800f-6b4d-44a1-a8cb-7445e2e03c5b)

**Comment by lukebakken** [comment]
I think you probably just need to be patient.

https://github.com/docker-library/rabbitmq/discussions/654

@tianon can you confirm?

**Reply by yosifkit** [reply, marked as answer]
Correct, we are still waiting for some jobs to run. We also merged a `debian` update yesterday so the queue is quite full.

As, for `3.12-management-alpine` having more architectures than `3.12.8-management-alpine`, that's because it is a less specific tag so some of those architectures are still `3.12.7` (i.e. the newest `3.12` that is currently ready).  This helps to ensure that architectures don't just randomly disappear on every version update on the less specific tags when some architectures build faster/slower than others (or queues slow them down).

**Comment by whalelines** [comment]
The builds seems to have completed successfully

-   https://github.com/docker-library/official-images/actions/runs/6711177894/job/18238008794
-   https://github.com/docker-library/meta/actions/runs/6714277978/job/18247259880

I see the image in oisupport/staging-amd64 https://hub.docker.com/layers/oisupport/staging-amd64/9cf9dcf2521d0d69550894096bc7a1ef7b4f79d112db5049a8409581608cf0b0/images/sha256-5d8463212afac9d21397a997703fc0f9c71fc5858589013b6341dfd89e408eca?context=explore

But I do not see the image in the arch-specific org: https://hub.docker.com/r/amd64/rabbitmq/tags

**Reply by lukebakken** [reply]
https://github.com/docker-library/rabbitmq/discussions/674#discussioncomment-7447976
